geom/alg_point_in_ring.go

```diff
@@ -10,9 +14,9 @@ const (
 	exterior side = +1
 )
 
-// pointRingSide checks the side of a ring that a point is on. It assumes that
+// relatePointToRing checks the side of a ring that a point is on. It assumes that
```

Comment:
We probably want to replace the implementation of this with a [winding number](https://en.wikipedia.org/wiki/Winding_number) algorithm instead. While it would have the same time complexity, it should have a much smaller constant (which can be verified using a benchmark). I'll create a ticket for that.